src/frontend/src/features/rooms/api/createRoom.ts

```diff
@@ -0,0 +1,25 @@
+import { useMutation, UseMutationOptions } from "@tanstack/react-query";
+import { fetchApi } from '@/api/fetchApi';
+import { ApiError } from "@/api/ApiError";
+import { ApiRoom } from "./ApiRoom";
+
+export interface CreateRoomParams {
+  slug: string;
+}
+
+const _createRoom = async ({slug}: CreateRoomParams): Promise<ApiRoom> => {
```

Comment:
why the `_`? :)

Reply:
is the `async/await` boilerplate necessary there? Can't we just `return fetchApi` ?